src/AppBundle/Controller/Api/AccountController.php

```diff
@@ -63,6 +64,24 @@ public function loginAction(Request $request)
         return new JsonResponse($userDto);
     }
 
+    /**
+     * @Route("api/account/logout")
```

Comment:
Annotations er deprecated.
<img src="https://user-images.githubusercontent.com/46197518/97871003-1d1e2b80-1d14-11eb-8d03-1d4513dcf8c9.png" width=350>

Forslag til endring:
[StackOverflow link](https://stackoverflow.com/questions/51171934/how-to-fix-symfony-3-4-route-and-method-deprecation)

Reply:
https://medium.com/@nebkam/symfony-deprecated-route-and-method-annotations-4d5e1d34556a